docs/detections/rules-ui-manage.asciidoc

```diff
@@ -135,9 +135,11 @@ Similarly, rules will be skipped if they can't be modified by a bulk edit. For e
 * Bulk edit multiple rules: Select the rules you want to edit, then select an action from the *Bulk actions* menu:
 ** *Index patterns*: Add or delete the index patterns used by all selected rules.
 ** *Tags*: Add or delete tags on all selected rules.
-** *Add rule actions*: Add <<rule-notifications,notification actions>> on all selected rules.
+** *Add rule actions*: Add <<rule-notifications,notification actions>> on all selected rules. The action frequency you select applies to all actions (both new and existing) on all selected rules. If you don't want to change the frequency of existing actions, update the rules separately.
```

Comment:
This does not feel correct to me. In bulk editing you can add actions and select individual frequencies for each of them. Those actions will be added to all selected rules after that.

Reply:
```suggestion
** *Add rule actions*: Add <<rule-notifications,rule actions>> on all selected rules. If you add multiple actions, you can specify action frequency for each action. If you don't want to change the frequency of existing actions on certain rules, update those rules separately.
```